Platform/Intel/MinPlatformPkg/PlatformInit/PlatformInitPei/FspSupport.c

```diff
@@ -1,3 +1,10 @@
+/** @file FspSupport.c
+  Provides FSP mode selection value based on PcdFspModeSelection
+
+Copyright (c) Microsoft Corporation.

```

Comment:
typically we will add start year in copyright for new files, is this intended to be no year at all?

Reply:
> typically we will add start year in copyright for new files, is this intended to be no year at all?

yes, below is one example from MdeModulePkg

https://github.com/tianocore/edk2/blob/master/MdeModulePkg/MdeModulePkg.dec#L11